recipes/dask-ms/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dask-ms" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 40a91034e53e2efb9b289eb5d9475a1957e1bd4dee707d133c3120990c5658d1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - dask
+    - numpy
+    - python-casacore
+    - setuptools
+    - xarray
+  run:
```

Comment:
The `setup.py` specifies version restrictions for several of the packages, so it is probably advisable to duplicate those restrictions here:

https://github.com/ska-sa/dask-ms/blob/master/setup.py